langchain/src/agents/executor.ts

```diff
@@ -79,20 +83,28 @@ export class AgentExecutor extends BaseChain {
       if (this.returnIntermediateSteps) {
         return { ...returnValues, intermediateSteps: steps, ...additional };
       }
-      await this.callbackManager.handleAgentEnd(finishStep, this.verbose);
+      await this.callbackManager.handleAgentEnd(
```

Comment:
Shouldn't this now be just callbackManager. ?

Reply:
yup good catch